test/SerialIOTest.cpp

```diff
@@ -2402,6 +2402,35 @@ TEST_CASE( "git_hdf5_sample_fileBased_read_test", "[serial][hdf5]" )
     }
 }
 
+TEST_CASE( "git_hdf5_early_chunk_query", "[serial][hdf5]" )
+{
+    try
+    {
+        Series s = Series(
+            "../samples/git-sample/data%T.h5",
+            Access::READ_ONLY
+        );
+
+        auto electrons = s.iterations[400].particles["electrons"];
```

Comment:
tried also this:
```suggestion
        auto it = s.iterations[400];
        auto electrons = it.particles["electrons"];
        it.open(); // or here...
```
was surprised this did not work, IIRC

Reply:
works for me

Reply:
Ok, cool! Thanks, then I must have done something wrong or mixed it up in my mind :tada: